stack_orchestrator/data/stacks/uniswap-frontend/stack.yml

```diff
@@ -0,0 +1,13 @@
+version: "0.1"
+name: uniswap-frontend
+repos:
+  # Uniswap interface
+  # TODO: Use release
+  - github.com/cerc-io/uniswap-interface@pm-urbit-hosting
+containers:
+  - cerc/uniswap-interface
+  # TODO: Add container for urbit
+pods:
+  - uniswap-interface
+  - urbit
+  # TODO: Add pod for urbit
```

Comment:
dockerizing urbit is going to be tricky and is a separate (lower priority) project, and also a separate SO stack